pax-web-tomcat/src/main/java/org/ops4j/pax/web/service/tomcat/internal/OsgiContextConfiguration.java

```diff
@@ -357,6 +227,147 @@ public void lifecycleEvent(LifecycleEvent event) {
 				Thread.currentThread().setContextClassLoader(tccl);
 			}
 
+			LoginConfig lc;
+			boolean noAuth = false;
+
+			if (loginConfig == null) {
+				authenticationValve = null;
+				lc = new LoginConfig("NONE", null, null, null);
+				noAuth = true;
+			} else {
+				// see org.apache.catalina.startup.ContextConfig.authenticatorConfig()
+
+				lc = new LoginConfig(loginConfig.getAuthMethod(), loginConfig.getRealmName(),
+						loginConfig.getFormLoginPage(), loginConfig.getFormErrorPage());
+
+				// Has an authenticator been configured already?
+				authenticationValve = (Valve) context.getAuthenticator();
+				if (authenticationValve == null) {
```

Comment:
OK @stsiano I didn't spot this ;) It's hard to analyze move+change diffs ;)
Some people like to squash everything, I like more commits which show the intentions ;)

But no problem here at all - :+1: .